src/components/layout.js

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { useStaticQuery, Link, graphql } from "gatsby"
+import styles from "./styles/layout.module.css";
+
+export default function Layout(props) {
+
+    const data = useStaticQuery(
+        graphql
+        `
+        query {
+            site {
+            siteMetadata {
+                title
+            }
+            }
+        }
+        `
+    );
+
+    return (
+    <div
+    className={styles.layoutBox}
+    >
```

Comment:
Typically properties in JS components are either:

- inline if they're short (`<div className={styles.layoutBox}>`)
- indented if they're long:

```js
<div 
  className={styles.layoutBox}
  onClick={foo}
  style={{ height: '100px }}
>
```